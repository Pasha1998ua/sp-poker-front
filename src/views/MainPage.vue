<template>
  <div class="main-page">
    <UserNameControl/>
    <RoomControl/>
    <Chat/>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue'
import Chat from '@/components/Chat.vue'
import RoomControl from '@/components/RoomControl.vue'
import UserNameControl from '@/components/UserNameControl.vue'

import localStorageHelper from '@/helpers/localStorageHelper'

export default defineComponent({
  name: 'MainPage',
  components: {
    Chat,
    RoomControl,
    UserNameControl
  },
  data () {
    return {}
  },
  created () {
    localStorageHelper.getUserId()
  }
})
</script>

<style scoped lang="scss">
</style>
