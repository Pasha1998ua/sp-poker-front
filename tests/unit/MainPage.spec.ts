import { shallowMount } from '@vue/test-utils'
import MainPage from '@/views/MainPage.vue'

describe('MainPage.vue', () => {
  it('renders page correctly', () => {
    const wrapper = shallowMount(MainPage)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
