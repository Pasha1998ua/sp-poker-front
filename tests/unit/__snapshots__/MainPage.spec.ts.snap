// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage.vue renders page correctly 1`] = `
<div class="main-page">
  <user-name-control-stub></user-name-control-stub>
  <room-control-stub></room-control-stub>
  <chat-stub></chat-stub>
</div>
`;
